README.md

```diff
@@ -432,6 +432,8 @@ The Direct Answer Component will render the BEST result, if found, based on the
 ANSWERS.addComponent('DirectAnswer', {
   // Required, the selector for the container element where the component will be injected
   container: '.direct-answer-container',
+  // Optional, a custom card component to use.
+  defaultCard: 'MyCustomCard',
```

Comment:
this being called defaultCard is kind of confusing to me. why not card or customCard?

Reply:
I'll post in the spec I was wondering as well

Reply:
Yeah, defaultCard is definitely not the best name. Maybe answerCardType or something like that? I'm hesitant to call it cardType since we don't want people thinking they can plug results cards into here.

Reply:
pinged rose still waiting, will ping again soon if no response

Reply:
https://yext.slack.com/archives/GREFVUD7V/p1591712960014200

tldr: we want to call it defaultCard because in the future there are plans to add cardOverrides